oak_attestation_verification/src/verifier.rs

```diff
@@ -720,6 +714,82 @@ fn verify_measurement_digest(
     }
 }
 
+/// Adapted version of verify_measure_digest() for kernels.
+fn verify_kernel_measurement_digest(
+    image_measurement: &RawDigest,
+    setup_data_measurement: &RawDigest,
+    now_utc_millis: i64,
+    endorsement: Option<&TransparentReleaseEndorsement>,
+    reference_value: &KernelBinaryReferenceValue,
+) -> anyhow::Result<()> {
+    let actual_image = raw_to_hex_digest(image_measurement);
```

Comment:
I don't think we should convert evertyhing to hex digests.

Reply:
Will fix as part of b/328732630.